libs/zzz/ui/src/Character/CharacterProfilePieces.tsx

```diff
@@ -150,3 +152,177 @@ function LevelBadge({
     </Typography>
   )
 }
+
+export function CharacterCoverOptimize({
+  character: { level, promotion, key: characterKey },
+}: {
+  character: ICachedCharacter
+}) {
+  const character = getCharStat(characterKey)
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', width: '500px' }}>
+      <CharImage characterKey={characterKey} character={character} />
+      <CharInformation
+        characterKey={characterKey}
+        character={character}
+        promotion={promotion}
+        level={level}
+      />
+    </Box>
+  )
+}
+
+function CharImage({
+  characterKey,
+  character,
+}: {
+  characterKey: CharacterKey
+  character: CharacterData
+}) {
+  const { attribute, id } = character
+  return (
+    <Box
+      sx={(theme) => ({
+        width: '100%',
+        height: '100%',
+        position: 'relative',
+        display: 'flex',
+        background: `${theme.palette[attribute].main}`,
+        zIndex: 1,
+      })}
+    >
+      <Box
+        sx={() => ({
+          position: 'absolute',
+          overflow: 'hidden',
```

Comment:
remove `overflow:hidden` to not clip text in the rotated corner
![image](https://github.com/user-attachments/assets/aff74602-d77b-478f-885a-d1c5ee0198d0)